notebook/static/notebook/js/maintoolbar.js

```diff
@@ -55,10 +55,34 @@ define([
             ],
             'run_int'],
          ['<add_celltype_list>'],
-         [['jupyter-notebook:show-command-palette']]
+         [['jupyter-notebook:show-command-palette']],
+         ['<add_celltoolbar_reminder>']
         ];
         this.construct(grps);
     };
+   
+    MainToolBar.prototype._pseudo_actions = {};
+
+
+    // reminder of where the celltoolbar new menu is, remove for 5.0
+    MainToolBar.prototype._pseudo_actions.add_celltoolbar_reminder = function () {
+        var _b = $('<button/>').attr('title','show new celltoolbar selector location').addClass('btn btn-default').text('CellToolbar')
+        var btn = $('<div/>').addClass('btn-group').append(_b)
+
+        _b.on('click', function(){
+            setTimeout(function(){$('#view_menu').parent().addClass('pulse')},0) 
+            setTimeout(function(){$('#view_menu').parent().addClass('open')},1000) 
+            setTimeout(function(){$('#menu-cell-toolbar').children('a').addClass('pulse')},2000)
+            setTimeout(function(){$('#menu-cell-toolbar').children('ul').css('display','block')},3000)
+            setTimeout(function(){$('#menu-cell-toolbar').children('ul').css('display','')},5400)
+            setTimeout(function(){$('#menu-cell-toolbar').children('a').removeClass('pulse')},5600)
+            setTimeout(function(){$('#view_menu').parent().removeClass('open')},5800)
+            setTimeout(function(){$('#view_menu').parent().removeClass('pulse')},6000)
```

Comment:
Yes that "remove" part is faster. I played a bit for the menu appearance to look "natural", and for that the disappear part need to be faster than the appear part (like when you remove your mouse).

If you look closely, each step of opening the menu is 1s, it stays open for 2.4 and disappear by step of 200ms.